catalog/app/utils/GraphQL/Provider.tsx

```diff
@@ -108,6 +108,7 @@ export default function GraphQLProvider({ children }: React.PropsWithChildren<{}
             p.bucket?.name && p.policy?.id ? `${p.bucket.name}/${p.policy.id}` : null,
           RoleBucketPermission: (p: any) =>
             p.bucket?.name && p.role?.id ? `${p.bucket.name}/${p.role.id}` : null,
+          SsoConfig: (c) => c.timestamp as string,
```

Comment:
is it actually a string? should be a Date afaict

Reply:
Looks like string 
![image](https://github.com/user-attachments/assets/d609702a-8bd2-4c85-b6dd-6654897812ff)

Reply:
you are showing the raw response -- that's not what gets into the system. it's parsed and processed by the urql machinery. try adding a `console.log` to the key function instead to see the actual data being used.

Reply:
Yes, this is a Date.
So, what would be more appropriate? Something like this?

```diff
diff --git a/catalog/app/utils/GraphQL/Provider.tsx b/catalog/app/utils/GraphQL/Provider.tsx
index e1f1414d..eb08796e 100644
--- a/catalog/app/utils/GraphQL/Provider.tsx
+++ b/catalog/app/utils/GraphQL/Provider.tsx
@@ -108,7 +108,7 @@ export default function GraphQLProvider({ children }: React.PropsWithChildren<{}
             p.bucket?.name && p.policy?.id ? `${p.bucket.name}/${p.policy.id}` : null,
           RoleBucketPermission: (p: any) =>
             p.bucket?.name && p.role?.id ? `${p.bucket.name}/${p.role.id}` : null,
-          SsoConfig: (c) => c.timestamp as string,
+          SsoConfig: (c) => (c.timestamp ? c.timestamp.toString() : null),
           Status: () => null,
           StatusReport: (r) => (typeof r.timestamp === 'string' ? r.timestamp : null),
           StatusReportList: () => null,
```

Reply:
yeah, tho i'd prefer `.toISOString()` or even `.getTime().toString()`

Reply:
> .getTime().toString()

Yeah, I tried `valueOf()` first, but the string is required

Reply:
Actually, maybe, `.valueOf().toString()`? Because, if I use `.getTime()` I should do type coercing `(c.timestamp as Date).getTime().toString()`

Reply:
Though, `as Date` here could be seen as more explicit code

Reply:
```tsx
           SsoConfig: (c) =>
            c.timestamp instanceof Date ? c.timestamp.getTime().toString() : null,
```